netlify.toml

```diff
@@ -1,5 +1,5 @@
 [[redirects]]
-from = "https://sheepdogjs.com/"
-to = "https://sheepdog.run/"
+from = "/*"
```

Comment:
Wouldn't this also apply to sheepdog.run?

Reply:
To be honest, I'm not sure. I couldn't really get a clear answer from the netlify docs so thought I'd just try it out. It's probably safer to be specific though. I'll change it